extensions/ecg-dicom/package.json

```diff
@@ -1,11 +1,12 @@
 {
   "name": "@radicalimaging/ecg-dicom",
-  "version": "3.0.7",
+  "version": "5.1.0",
```

Comment:
The version is supposed to sync with the OHIF version, so this should be 3.2.8 or some such number.